client/invoke.go

```diff
@@ -0,0 +1,95 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+
+	v1 "github.com/dapr/go-sdk/dapr/proto/common/v1"
+	pb "github.com/dapr/go-sdk/dapr/proto/runtime/v1"
+	anypb "github.com/golang/protobuf/ptypes/any"
+	"github.com/pkg/errors"
+)
+
+// InvokeServiceWithRequest invokes service with input request
+func (c *Client) InvokeServiceWithRequest(ctx context.Context, req *pb.InvokeServiceRequest) (out []byte, err error) {
```

Comment:
this doesn't need to be exported. its just a helper method for other InvokeService* methods

Reply:
good catch, yes, this is only used internally